packages/core/package.json

```diff
@@ -85,10 +85,10 @@
     "@crackle/router": "0.1.2",
     "@rollup/plugin-commonjs": "^22.0.2",
     "@rollup/plugin-node-resolve": "^13.3.0",
-    "@vanilla-extract/babel-plugin": "^1.1.2",
-    "@vanilla-extract/css": "^1.7.3",
-    "@vanilla-extract/integration": "^5.0.1",
-    "@vanilla-extract/vite-plugin": "^3.3.1",
+    "@vanilla-extract/babel-plugin-debug-ids": "^1.0.0",
```

Comment:
Does this actually get used anywhere? Now that we're just calling `@vanilla-extract/integration#transform` directly